From db15be891c2ebc3a0341b79168b2ea0d0b8219ab Mon Sep 17 00:00:00 2001
From: chenyifu <chenyf@rock-chips.com>
Date: Wed, 14 Jan 2015 16:12:07 +0800
Subject: [PATCH] rk3368 mipi: modified the bug of dts file

The address of rk3368 is 64-bits.
Change the order of "pclk_mipi_dsi" and "pclk_mipi_dsi_host"
in order compatible with earlier code.

Signed-off-by: chenyifu <chenyf@rock-chips.com>
---
 arch/arm64/boot/dts/rk3368.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)
 mode change 100755 => 100644 arch/arm64/boot/dts/rk3368.dtsi

diff --git a/arch/arm64/boot/dts/rk3368.dtsi b/arch/arm64/boot/dts/rk3368.dtsi
old mode 100755
new mode 100644
index 6e1813281a03..25a73752d0cd
--- a/arch/arm64/boot/dts/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rk3368.dtsi
@@ -697,13 +697,13 @@
 	};
 
 	dsihost0: mipi@ff960000{
-		compatible = "rockchip,rk33x-dsi";
+		compatible = "rockchip,rk3368-dsi";
 		rockchip,prop = <0>;
-		reg = <0xff960000 0x4000>, <0xff968000 0x4000>;
+		reg = <0x0 0xff960000 0x0 0x4000>, <0x0 0xff968000 0x0 0x4000>;
 		reg-names = "mipi_dsi_host" ,"mipi_dsi_phy";
 		interrupts = <GIC_SPI 19 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&clk_gates4 14>, <&clk_gates17 3>, <&clk_gates22 10>;
-		clock-names = "clk_mipi_24m", "pclk_mipi_dsi_host", "pclk_mipi_dsi_phy";
+		clocks = <&clk_gates4 14>, <&clk_gates22 10>, <&clk_gates17 3>;
+		clock-names = "clk_mipi_24m", "pclk_mipi_dsi", "pclk_mipi_dsi_host";
 		status = "okay";
 	};
 
-- 
2.35.3

